yb-voyager/src/srcdb/pg_dump_extract_schema.go

```diff
@@ -12,16 +12,16 @@ import (
 	"github.com/yugabyte/yb-voyager/yb-voyager/src/utils"
 )
 
-func pgdumpExtractSchema(connectionUri string, exportDir string) {
+func pgdumpExtractSchema(source *Source, connectionUri string, exportDir string) {
 	fmt.Printf("exporting the schema %10s", "")
 	go utils.Wait("done\n", "error\n")
 
-	cmd := fmt.Sprintf(`pg_dump "%s" --schema-only --no-owner -f %s --no-privileges`,
-		connectionUri, filepath.Join(exportDir, "temp", "schema.sql"))
+	cmd := fmt.Sprintf(`pg_dump "%s" --schema-only --schema "%s" --no-owner -f %s --no-privileges`,
```

Comment:
Isn't `source.Schema` a pipe separated list?

Reply:
yes, pg_dump allows pipe separated list.
Actually `--schema` parameter in `pg_dump` takes in regex pattern for the schema list, if using `|` for joining schema names will serve the purpose for us.